transformer/kubernetes/apiresource/argocdapplication.go

```diff
@@ -70,6 +71,14 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
 		clusterServer = deployToSameCluster
 	}
 	appGVK := v1alpha1.ApplicationSchemaGroupVersionKind
+	ques := qaengine.FetchStringAnswer(
+		common.ConfigTransformersKubernetesArgoCDNamespaceKey,
+		"Enter destination namespace for argo cd pipeline",
```

Comment:
Can you please update the string to "Enter the destination namespace for the Argo CD pipeline"